spot_wrapper/wrapper.py

```diff
@@ -644,6 +647,8 @@ def __init__(
         self._last_trajectory_command_precise = None
         self._last_velocity_command_time = None
         self._last_docking_command = None
+        self._last_navigate_to_command = None
+        self._state_navigate_to_valid = None
```

Comment:
This might need to be changed if https://github.com/bdaiinstitute/spot_wrapper/pull/48 is merged.

Reply:
Move _last_navigate_to_command to `RobotCommandData` class.

Reply:
Rebase this branch to https://github.com/bdaiinstitute/spot_wrapper/pull/48.